lib/fluent/plugin/out_dynatrace.rb

```diff
@@ -64,7 +64,12 @@ def configure(conf)
         compat_parameters_convert(conf, :inject)
         super
 
+        raise Fluent::ConfigError, 'active_gate_url has not been set' if @active_gate_url == ''
```

Comment:
```suggestion
        raise Fluent::ConfigError, 'active_gate_url has not been set' if @active_gate_url.empty?
```
This would be the more common way in Ruby.

Would you also need to check for nil?